server/libs/userlibs.js

```diff
@@ -288,4 +288,178 @@ const uploadPhoto = function(req) {
     })
 };
 
-module.exports = {register, logIn, updateUser, logOut , changePassword, deleteUser, deActivateUser, getUser, uploadPhoto}
\ No newline at end of file
+const updateQuizProgress = function(data) {
+    return new Promise((resolve,reject) => {
+        if (data.answer) {
+            User.updateOne({uuid: data.uuid, "quizProgress.quizId" : data.quizId} , {$push : {"quizProgress.$.attempted" : data.questionId, "quizProgress.$.correct" : data.questionId}}, {new:true})
```

Comment:
do proper indentation when the line is exceeded from specific limit

Reply:
Done